Source/ACE.Server/WorldObjects/Gem.cs

```diff
@@ -77,10 +73,10 @@ public override void UseItem(Player player)
                     castMessage = "The gem casts Asheron's Lesser Benediction on you";
                 */
 
-                player.CreateItemSpell(Guid, (uint)SpellDID);
+                player.CreateItemSpell(this, (uint)SpellDID);
 
                 // always consume gem?
```

Comment:
There are instances of Gems that are perpetual, such as Blackmoor's Favor: https://pastebin.com/GaBckVtC.  According to the PCAP data, bool property 63 was used to denote unlimited usage.

Reply:
I'm not sure what you're getting at. Does a change need to be made?

Reply:
If the gem has PropertyBool.UnlimitedUse, as with the case of the Blackmoor's Favor gem that is given to all players since TOD, the gem needs to not be consumed on use, so it can be used over and over again.

Reply:
http://asheron.wikia.com/wiki/Blackmoor%27s_Favor